### PR TITLE
[ios] Add screen titles for the back button menu

### PR DESCRIPTION
**Pull request by kirylkaveryn**
This PR adds:
- screen titles for the back button menu (when a user long presses on the back button) to return on the proper screen
- new word `Map` in stings.txt

This PR doesn't affect devices earlier than iOS14.0 because the `minimal` back button style isn't available.
If we want to use this menu with titles we should use the `back button with title` for ios <14.0.

Tested on:
- iPhone 11Pro 17.2 (device)
- iPhone 6 12.5 (device) - doesn't affected
- iPad designed for macOS (device)
- iPhone 15pro  17.2 (simulator)

Before:
image

After:
image
image
image

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -29588,3 +29588,48 @@
...
+    ja = 地図
+    ko = 지도
+    lt = Žemėlapis
+    mr = नकाशा
+    nb = Kart
+    nl = Kaart
+    pl = Mapa
+    pt = Mapa
+    pt-BR = Mapa
+    ro = Hartă
```
Comment (vgrigoriu) [inline data/strings/strings.txt:29626]: :+1:

File: data/strings/strings.txt [hunk 2]
```diff
@@ -29588,3 +29588,48 @@
...
+    da = Kort
+    de = Karte
+    el = Χάρτης
+    es = Mapa
+    et = Kaart
+    eu = Mapa
+    fa = نقشه
+    fi = Kartta
+    fr = Carte
+    he = מפה
```
Comment (NeatNit) [inline data/strings/strings.txt:29612]: 👍 for `he`

**Review by matheusgomesms — APPROVED** [review]

**Review by d4f5409d — APPROVED** [review]
🇭🇺 seems ok ✅

**Review by jimcarst — APPROVED** [review]
The word 'map' is already translated in many places of the app. No need to bother the translators to check it.

**Comment by NeatNit** [comment]
> The word 'map' is already translated in many places of the app. No need to bother the translators to check it.

I just had a look (using Ctrl+F with "מפה", the Hebrew translation, for a shorter results list), and it's never on its own, so the new string is needed. The existence of other strings doesn't help, it still needs to be verified because the automatic translation might be wrong.

I agree that it's incredibly unlikely that any of the translations are wrong, but it still needs to be verified by the translators.

**Comment by kirylkaveryn** [comment]
> The word 'map' is already translated in many places of the app. No need to bother the translators to check it. (And, yes, it's ok for nl)

It's very hard to retrieve the particular word from sentences for over than 30 languages in correct grammar form. And easy to make a mistake...

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**